packages/neon-api/__tests__/provider/neoCli/core.ts

```diff
@@ -136,7 +136,7 @@ describe("getClaims", () => {
   test("returns successful claims", async () => {
     const testClaim = {
       end_height: 231,
-      generated: 1464,
+      generated: 0,

```

Comment:
Please use a non-zero unique number for `generated` and `sys_fee` since the code defaults to `0`.

Reply:
correction applied